tutorial/source/bayesian_regression.ipynb

```diff
@@ -70,7 +70,7 @@
     "    # b = 1\n",
     "    y = np.matmul(X, w) + np.repeat(1, N) + np.random.normal(0, noise_std, size=N)\n",
     "    y = y.reshape(N, 1)\n",
-    "    X, y = torch.tensor(X), torch.tensor(y)\n",
+    "    X, y = torch.tensor(X).type(torch.empty(1).type()), torch.tensor(y).type(torch.empty(1).type())\n",
```

Comment:
What does this weird trick do? Would it be possible to instead `.type(torch.Tensor)`?

Reply:
Yup, that's much cleaner :) I forgot that `torch.Tensor` just aliases the default tensor type. The reason we had to do this is because `torch.tensor(np.array([0.]))` gives a double tensor by default now.